cliff.toml

```diff
@@ -0,0 +1,99 @@
+# configuration file for git-cliff (0.1.0)
+
+[changelog]
+# changelog header
+header = """
+<!--
+Usage:
+
+Change log entries are generated by git cliff ref: https://github.com/orhun/git-cliff
+This can be run using "make changelog tag=vx.y.z"
+
+Each commit should be conventional, the following message groups are supported.
+
+* feat, feature
+* imp
+* bug, fix
+* deprecated
+* api-breaking
+* state-machine-breaking, smb
+
+Types of changes (Stanzas):
+
+"Features" for new features. (feat, feature)
+"Improvements" for changes in existing functionality. (imp)
+"Deprecated" for soon-to-be removed features.
+"Bug Fixes" for any bug fixes. (bug, fix)
+"API Breaking" for breaking exported APIs used by developers building on SDK. (api-breaking)
+"State Machine Breaking" for any changes that result in a different AppState given same genesisState and txList. (state-machine-breaking, smb)
+Ref: https://keepachangelog.com/en/1.0.0/
+-->
+
+# Changelog
+All notable changes to this project will be documented in this file.
+"""
+# template for the changelog body
+# https://tera.netlify.app/docs/#introduction
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | striptags | trim | upper_first }}
+    {% for commit in commits %}
+        * {{ commit.message | upper_first }}\
+    {% endfor %}
+{% endfor %}\n
+"""
+# remove the leading and trailing whitespace from the template
+trim = true
+# changelog footer
+footer = """
+<!-- generated by git-cliff -->
+"""
+
+[git]
+# parse the commits based on https://www.conventionalcommits.org
+conventional_commits = true
+# filter out the commits that are not conventional
+filter_unconventional = true
+# process each line of a commit as an individual commit
+split_commits = false
+# regex for preprocessing the commit messages
+commit_preprocessors = [
+    # A reference to an issue is appened to commits that looks like "(#1234)", this will be replaced
+    # with a link to that issue, e.g. "[#$1234](https://github.com/cosmos/ibc-go/issues/1234)".
+    { pattern = '\((\w+\s)?#([0-9]+)\)', replace = "([#${2}](https://github.com/cosmos/ibc-go/issues/${2}))" },
+    # any reference to a pr like "pr-1234" will be replaced with a link to the PR.
+    { pattern = '\(pr-([0-9]+)\)', replace = "([#${1}](https://github.com/cosmos/ibc-go/pulls/${1}))" },
+]
```

Comment:
this effectively means I can write a commit message like 

`fix: fixed the problem in issue #1234 and also addressed concerns raised in pr-5678`

And the issue and pr will be expanded into links in the changelog entry.

e.g. 

fix: fixed the problem in issue [#1234](https://github.com/cosmos/ibc-go/issues/1234) and also addressed concerns raised in [#5678](https://github.com/cosmos/ibc-go/pulls/5678)

Reply:
Awesome!

Reply:
Will the commit merged automatically be linked?

Reply:
@colin-axner yep the commit being merged will be linked. This is the issue number in parens at the end of the commit in the generated output.